lib/XML/LibXML/CStructs.pm

```diff
@@ -113,6 +119,27 @@ my class xmlError is export(:types) {
     has OpaquePointer $.node; # the node in the tree
 }
 
+my class xmlElement is export(:types) {
```

Comment:
Oh, I'm well aware of this after my initial experiences with Nativecall! =)

Looks like later versions of libxml2 will want $.contModel defined if LIBXML_REGEXP_ENABLED is set or not. This is actually a good thing!